sal_interfaces/GenericCamera/GenericCamera_Events.xml

```diff
@@ -794,4 +794,12 @@
     <Subsystem>GenericCamera</Subsystem>
     <EFDB_Topic>GenericCamera_logevent_ccsConfigured</EFDB_Topic>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>GenericCamera</Subsystem>
+    <EFDB_Topic>GenericCamera_logevent_startAutoExposure</EFDB_Topic>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>GenericCamera</Subsystem>
+    <EFDB_Topic>GenericCamera_logevent_stopAutoExposure</EFDB_Topic>
```

Comment:
How about `autoExposureStopped `?

Reply:
I modified it to `autoExposureStopped` but note that I was following the way the live view events have been implemented and there it is `endLiveView` (not stopLiveView) so we may want to modify that one as well.